rasa/core/channels/slack.py

```diff
@@ -166,14 +171,20 @@ def __init__(
                 If not set, messages will be sent back
                 to the "App" DM channel of your bot's name.
             proxy: A Proxy Server to route your traffic through
-            slack_retry_reason_header: Slack HTTP header name indicating reason that slack send retry request.
-            slack_retry_number_header: Slack HTTP header name indicating the attempt number
+            slack_retry_reason_header: Slack HTTP header name indicating reason
+                that slack send retry request.
+            slack_retry_number_header: Slack HTTP header name indicating
+                the attempt number
             errors_ignore_retry: Any error codes given by Slack
                 included in this list will be ignored.
                 Error codes are listed
                 `here <https://api.slack.com/events-api#errors>`_.
-            use_threads: If set to True, your bot will send responses in Slack as a threaded message.
-                Responses will appear as a normal Slack message if set to False.
+            use_threads: If set to True, your bot will send responses in Slack as
+                a threaded message. Responses will appear as a normal Slack message
+                if set to False.
+            slack_signing_secret: Slack creates a unique string for your app and
+                shares it with you. This allows us to verify requests from Slack
+                with confidence by verifying signatures using your signing secret.
```

Comment:
optional: what does the `with confidence` part mean?

Reply:
copied this from the slack docs :joy: